templates/qaoa.yaml

```diff
@@ -17,19 +21,83 @@ spec:
             from zquantum.qaoa.maxcut import get_maxcut_hamiltonian
             from qeopenfermion import save_qubit_operator
             graph = load_graph('graph.json')
-            hamiltonian = get_maxcut_hamiltonian(graph)
+            hamiltonian = get_maxcut_hamiltonian(graph, scaling={{inputs.parameters.scaling}}, shifted={{inputs.parameters.shifted}})
             save_qubit_operator(hamiltonian,'hamiltonian.json')
     outputs:
       artifacts:
       - name: hamiltonian
         path: /app/hamiltonian.json
 
-  - name: solve-maxcut-by-exhaustive-search
```

Comment:
Seems to me that you've removed `solve-maxcut-by-exhaustive-search` template alltogether or am I missing something?

Reply:
Yes...It has been restored in f0ccbb3d0774bdfaae755c22a3ebd985feaa9d86